csrc/trtllm_gemm_runner.cu

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+#include <ATen/cuda/EmptyTensor.h>
+#include <ATen/native/cuda/Resize.h>
+#include <Python.h>
```

Comment:
It should have already been included in `pytorch_extension_utils.h` so no need to include it again.

Reply:
Thanks, removed 2 headers, the `EmptyTensor.h` is still necessary to use `at::detail::empty_cuda`. I see the same for the csrc\trtllm_fused_moe_kernel_launcher.cu.